scripts/map_labels.py

```diff
@@ -62,10 +62,25 @@ def map_vessels2other(name: str):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Map labels.')
     parser.add_argument('-i', '--input_dir', dest='in_dir', type=str, required=True, help='input directory')
-    parser.add_argument('-o', '--output_dir', dest='out_dir', type=str, help='output directory')
+    parser.add_argument('-o', '--output_dir', dest='out_dir', type=str, required=True, help='output directory')
```

Comment:
I need to persuade you to start using [typer](https://typer.tiangolo.com/#example)
Basically, from the annotations in a function , the decorator can build a CLI for you. Here it would be something like

```python
import typer
from pathlib import Path
from typing import Optional

def main(input_dir: Path, output_dir: Path, input_tissues: Optional[str], output_tissues: str):
   ...
   if input_tissues:
           ...
if __name__ == "__main__":
    typer.run(main)
```

Reply:
I tried typer but did not fall in love (yet)